scripts/gathering/CatalystsAndGemsSnapshot.ts

```diff
@@ -0,0 +1,46 @@
+import {outputJsonSync} from 'fs-extra';
+import {ethers} from 'hardhat';
+const fromBlock = 0;
+
+void (async () => {
+  console.log(
+    'Current block',
+    (await ethers.provider.getBlock('latest')).number
+  );
+  await listEvents([
+    {
+      contractName: 'OldCatalystRegistry',
+      eventNames: ['CatalystApplied', 'GemsAdded'],
+      toBlock: 7370475,
```

Comment:
why this block number?

Reply:
Thats the first event block on goerli for that contract, I guess it will be fine to leave it as 0 too, it's just 88 events

Reply:
Sorry, disregard last message, thats the block the new contract was deployed, newer mints are not taken into account since new assets are not being migrated to the new contract